app/Console/Kernel.php

```diff
@@ -15,7 +18,20 @@ class Kernel extends ConsoleKernel
      */
     protected function schedule(Schedule $schedule)
     {
-        // $schedule->command('inspire')->hourly();
+        $schedule->call(function () {
+            $cart_ids = Cart::all()->value('id');
+            $cart_ids_array = [];
+            array_push($cart_ids_array, $cart_ids);
+            foreach ($cart_ids_array as $cart_id) {
+                $last_updated_at_record = DB::table('cart_details')->where('cart_id', '=', $cart_id)->orderBy('updated_at', 'desc')->first();
+                $first = strtotime(now());
+                $last = strtotime($last_updated_at_record->updated_at);
+
+                if ($first - $last >= 60) {
+                    (new CartController)->clear($cart_id);
+                }
+            }
+        })->everyMinute();
```

Comment:
- Better add a [Job](https://laravel.com/docs/9.x/queues#dispatching-jobs) to handle the logic
- Your code will cause N+1 query problem, query could be done once to fetch all data needed